third-party/chpl-venv/chpldoc-requirements1.txt

```diff
@@ -0,0 +1 @@
+MarkupSafe==2.0.1
```

Comment:
Can we add a comment to these three files to help understand why the division was made?